src/screens/GroupInfo/index.js

```diff
@@ -27,6 +27,7 @@ import ModalActionAnonymous from './elements/ModalActionAnonymous';
 import ReportGroup from '../../assets/images/report.png';
 import useGroupInfo from './hooks/useGroupInfo';
 import {CHANNEL_TYPE_ANONYMOUS} from '../../utils/constants';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have imported `COLORS` from `'../../utils/theme'`, but there is also an import statement for `colors` from `'../../utils/colors'`. If these two are the same, consider removing one to avoid confusion and redundancy. If they are different, consider renaming one of them to make it clear what the difference is.
```diff
- import {COLORS} from '../../utils/theme';
- import {colors} from '../../utils/colors';
+ import {THEME_COLORS} from '../../utils/theme';
+ import {UTIL_COLORS} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->